### ISSUE TITLE
Improved isClosed()

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Map#266]
knowing that Map is closed is pretty important because the JVM segfaults if you call remainingAutoResizes() and the map has been closed

**Comment by RobAustin** [comment]
we have added the following test 

```
    @Test(expected = ChronicleHashClosedException.class)
    public void testRemainingAutoResizesAfterClose() {
        ChronicleMap<Integer, Integer> map = of(Integer.class, Integer.class).entries(1).create();
        map.close();
        map.remainingAutoResizes();
    }
```

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.21ea3](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.21ea3), [BOM-2.21ea37](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.21ea37)
